packages/flame/lib/src/game/game_render_box.dart

```diff
@@ -2,14 +2,52 @@ import 'package:flame/src/game/game_loop.dart';
 import 'package:flame/src/game/mixins/game.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart' hide WidgetBuilder;
-//ignore_for_file: unnecessary_non_null_assertion
+
+class RenderGameWidget extends LeafRenderObjectWidget {
```

Comment:
This class is only used by `GameWidget`, so perhaps move it into `game_widget.dart` and make private?

Reply:
This file is not part of the public API.

Also `GameWidget` being the only user of this widget is circumstantial as we should have their concerns as separated as possible. IMO having them in different files illustrates this,.

Reply:
But now that the class is public, its purpose and function has to be documented.